src/Pim/Bundle/CatalogBundle/tests/integration/ElasticSearch/Filter/search_specification.rst

```diff
@@ -898,16 +898,13 @@ Same as ``IN``, but with the ``must_not`` occured type
 
 Family
 ******
-:Apply: "family" field
+:Apply: apply on 'family' field
```

Comment:
Ok, I correct, can I correct for the category too ? -> :Apply: apply on 'categories' field